Frends.OAuth.CreateJWTToken/Frends.OAuth.CreateJWTToken/Definitions/Result.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace Frends.OAuth.CreateJWTToken.Definitions;
+
+/// <summary>
+/// Return token.
+/// </summary>
+public class Result
+{
+    /// <summary>
+    /// Return token.
+    /// </summary>
+    public List<TokenResult> Results { get; internal set; }
```

Comment:
You only create one token in the task, so you don't have to return a list as a Result. One `TokenResult` is enough, so this class can be removed.